functions.yml

```diff
@@ -407,16 +428,17 @@ gwas_functions:
           [Row(beta=1.1658962684583645, oddsRatio=3.208797540870915, waldConfidenceInterval=[0.2970960052553798, 34.65674891673014], pValue=0.2943946848756771)]
       args:
         - name: genotypes
-          doc: An array of genotypes
+          doc: An array of genotype structs
         - name: phenotypes
-          doc: An array of phenotype values
+          doc: A double array of phenotype values
         - name: covariates
-          doc: a matrix of covariates
+          doc: A ``spark.ml`` ``Matrix`` of covariates
         - name: test
           doc: Which logistic regression test to use. Can be 'LRT' or 'Firth'
           type: str
+      returns: A struct containing ``beta``, ``oddsRatio``, ``waldConfidenceInterval``, and ``pValue`` fields. See :ref:`logistic-regression`.
     - name: genotype_states
-      doc: Get number of alt alleles for a genotype
+      doc: Gets number of alternate alleles for genotypes. Returns ``-1`` if there are any ``-1`` s in the calls array.
```

Comment:
`Gets number of alternate alleles for genotypes.` -> `Gets the number of alternate alleles for an array of genotype structs.` We should also clarify that `-1` represents a missing call.

Reply:
Done